multiwallet.go

```diff
@@ -43,7 +43,9 @@ type MultiWallet struct {
 
 	Politeia  *Politeia
 	dexClient *DexClient
-	VspList   []*VSPInfo
+
+	vspMu sync.RWMutex
+	vsps  []*VSP
```

Comment:
This does not look like it is used anywhere.

Reply:
artefact, will remove

Reply:
eh, no, they're used in the vsp.go file